addons/website_forum/views/website_forum.xml

```diff
@@ -19,10 +19,10 @@
     <form  t-attf-method="#{form_method or 'POST'}" t-att-action="url" t-attf-class="#{form_classes} #{not inDropdown and 'btn btn-sm border'}">
         <input type="hidden" name="csrf_token" t-att-value="request.csrf_token()"/>
         <div class="input-group">
-            <a t-attf-class="#{icon and not label and ('fa ' + icon)} #{inDropdown and 'dropdown-item pl-3' or 'btn btn-sm p-0'} #{classes} #{karma and 'karma_required text-muted'}" t-attf-data-karma="#{karma}" t-att-title="title">
+            <button t-attf-class="#{icon and not label and ('fa ' + icon)} #{inDropdown and 'dropdown-item pl-3' or 'btn btn-sm p-0'} #{classes} #{karma and 'karma_required text-muted'}" t-attf-data-karma="#{karma}" t-att-title="title">
```

Comment:
This is breaking the layout, the `button` was changed to a `a` in last minute before r+ to fix an ugly black border around it.

![image](https://user-images.githubusercontent.com/30048408/145241534-01acce9b-1ed4-48b6-a62f-d174ecd5a3fb.png)

Reply:
I've added a CSS rule to remove the outline.

I also have created a task with Framework JS because the text becomes white when clicking on the item (but it's also the case in a few other places: https://www.odoo.com/web#id=2711542&cids=1&menu_id=4720&action=333&active_id=133&model=project.task&view_type=form )